### PR TITLE
Added toRaster test.

### PR DESCRIPTION
**Pull request by emilwestergren**
* Tests converting `GpuImage` to `RasterImage`

@sebastianbaginski peer review

### CODE REVIEW

**Comment by sebastianbaginski** [comment]
Lack of serious generic programming facilities in ooc makes me sad, it could be one method parametrized on image type :(

**Comment by emilwestergren** [comment]
Removed duplicated code.

**Comment by sebastianbaginski** [comment]
OK, this one is nice too :)

File: test/draw/gpu/ToRasterTest.ooc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		gpuImage := context createImage(sourceImage)
+		raster := gpuImage toRaster()
+		expect(raster distance(sourceImage), is equal to(0.0f))
+		sourceImage free()
+		gpuImage free()
+		raster free()
+	}
+	init: func {
+		super("ToRaster")
+		this add("toRaster bgra", || this toRasterTestFunction(RasterBgra open("test/draw/gpu/input/Flower.png")))
```
Comment (marcusnaslund) [inline test/draw/gpu/ToRasterTest.ooc]: This is a new way of writing tests, to be sure. I won't object to it, though, as it looks pretty nice.

**Comment by davidpiuva** [comment]
This PR failed the license test and was merged anyway.

**Comment by marcusnaslund** [comment]
My bad for taking shortcuts, I didn't want to wait on all the vidproc builds when all this PR did was change kean tests. I failed to look at the license checker printout. Anyway, I've fixed upstream now.

**Pull request merged**